person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
[ OPTIONAL CHANGE ]

- [ ] The previous reviewer did not highlight this, but you forgot to add a method to add_rental in your Person class. Please consider adding this method. 

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
@AmaduKamara 
thanks, I will